app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -186,6 +186,13 @@ define(function (require, exports, module) {
         .then(_.bind(this._notifyProfileUpdate, this, account.get('uid')));
     },
 
+    updateDisplayEmail (email) {
+      var account = this.getSignedInAccount();
+      account.set('email', email);
+      return this.user.setAccount(account)
+        .then(_.bind(this._notifyProfileUpdate, this, account.get('uid')));
```

Comment:
I realize `updateDisplayName` does it this way, it seems cleaner to do something like:

```js
  .then(() => this._notifyProfileUpdate(account.get('uid')));
```